src/components/Hero.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import '../index.css';
+
+const Hero = ({ title, subtitle, buttonText, onClick, heroImage }) => {
```

Comment:
The hero section looks nice, just a bit of styling is needed for the button :-) According to Wave there is too little color contrast between the background image and the text but to me, it seems fine.